vaadin-ordered-layout-flow-parent/vaadin-ordered-layout-flow/src/main/java/com/vaadin/flow/component/orderedlayout/HorizontalLayout.java

```diff
@@ -55,6 +55,41 @@ public HorizontalLayout(Component... children) {
         add(children);
     }
 
+    /**
+     * Convenience constructor to create a layout with the children already
```

Comment:
Nit (opinionated comment): would rephrase to `...with children and specified justifyContentMode.`